<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ControlSet</class>
 <widget class="QWidget" name="ControlSet">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1500</width>
    <height>800</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>509</x>
     <y>133</y>
     <width>171</width>
     <height>30</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>黑体</family>
     <pointsize>20</pointsize>
    </font>
   </property>
   <property name="text">
    <string>控制设置</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QPushButton" name="tendency_Btn">
   <property name="geometry">
    <rect>
     <x>700</x>
     <y>120</y>
     <width>60</width>
     <height>60</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">QPushButton{border-image: url(:/new/prefix1/source/treatMode/tendencyNormal.png);}
QPushButton:hover{border-image: url(:/new/prefix1/source/treatMode/tendencyNormalHover.png);}</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QPushButton" name="displacement_Btn">
   <property name="geometry">
    <rect>
     <x>810</x>
     <y>120</y>
     <width>60</width>
     <height>60</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">QPushButton{border-image: url(:/new/prefix1/source/treatMode/displacementNormal.png);}
QPushButton:hover{border-image: url(:/new/prefix1/source/treatMode/displacementHover.png);}</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QRadioButton" name="thumb_RadioButton">
   <property name="geometry">
    <rect>
     <x>700</x>
     <y>230</y>
     <width>121</width>
     <height>40</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>黑体</family>
     <pointsize>20</pointsize>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">QRadioButton::indicator::unchecked {width: 32px;height: 18px;image:url(&quot;:/new/prefix1/source/buttons/generalButton/radioNormal.png&quot;);position:relative;left: 0px;right: 5px;top: 2px;bottom:4px;}
QRadioButton::indicator::checked {width: 32px;height: 18px;image:url(&quot;:/new/prefix1/source/buttons/generalButton/radioSelected.png&quot;);position:relative;left: 0px;right: 5px;top: 2px;bottom:4px;}</string>
   </property>
   <property name="text">
    <string>拇指</string>
   </property>
   <property name="checked">
    <bool>true</bool>
   </property>
   <property name="autoExclusive">
    <bool>false</bool>
   </property>
  </widget>
  <widget class="QRadioButton" name="foreFinger_RadioButton">
   <property name="geometry">
    <rect>
     <x>700</x>
     <y>286</y>
     <width>111</width>
     <height>40</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>黑体</family>
     <pointsize>20</pointsize>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">QRadioButton::indicator::unchecked {width: 32px;height: 18px;image:url(&quot;:/new/prefix1/source/buttons/generalButton/radioNormal.png&quot;);position:relative;left: 0px;right: 5px;top: 2px;bottom:4px;}
QRadioButton::indicator::checked {width: 32px;height: 18px;image:url(&quot;:/new/prefix1/source/buttons/generalButton/radioSelected.png&quot;);position:relative;left: 0px;right: 5px;top: 2px;bottom:4px;}</string>
   </property>
   <property name="text">
    <string>食指</string>
   </property>
   <property name="checked">
    <bool>true</bool>
   </property>
   <property name="autoExclusive">
    <bool>false</bool>
   </property>
  </widget>
  <widget class="QRadioButton" name="middleFinger_RadioButton">
   <property name="geometry">
    <rect>
     <x>700</x>
     <y>346</y>
     <width>101</width>
     <height>40</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>黑体</family>
     <pointsize>20</pointsize>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">QRadioButton::indicator::unchecked {width: 32px;height: 18px;image:url(&quot;:/new/prefix1/source/buttons/generalButton/radioNormal.png&quot;);position:relative;left: 0px;right: 5px;top: 2px;bottom:4px;}
QRadioButton::indicator::checked {width: 32px;height: 18px;image:url(&quot;:/new/prefix1/source/buttons/generalButton/radioSelected.png&quot;);position:relative;left: 0px;right: 5px;top: 2px;bottom:4px;}</string>
   </property>
   <property name="text">
    <string>中指</string>
   </property>
   <property name="checked">
    <bool>true</bool>
   </property>
   <property name="autoExclusive">
    <bool>false</bool>
   </property>
  </widget>
  <widget class="QRadioButton" name="ringFinger_RadioButton">
   <property name="geometry">
    <rect>
     <x>700</x>
     <y>406</y>
     <width>131</width>
     <height>40</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>黑体</family>
     <pointsize>20</pointsize>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">QRadioButton::indicator::unchecked {width: 32px;height: 18px;image:url(&quot;:/new/prefix1/source/buttons/generalButton/radioNormal.png&quot;);position:relative;left: 0px;right: 5px;top: 2px;bottom:4px;}
QRadioButton::indicator::checked {width: 32px;height: 18px;image:url(&quot;:/new/prefix1/source/buttons/generalButton/radioSelected.png&quot;);position:relative;left: 0px;right: 5px;top: 2px;bottom:4px;}</string>
   </property>
   <property name="text">
    <string>无名指</string>
   </property>
   <property name="checked">
    <bool>true</bool>
   </property>
   <property name="autoExclusive">
    <bool>false</bool>
   </property>
  </widget>
  <widget class="QRadioButton" name="littleFinger_RadioButton">
   <property name="geometry">
    <rect>
     <x>700</x>
     <y>466</y>
     <width>101</width>
     <height>40</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>黑体</family>
     <pointsize>20</pointsize>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">QRadioButton::indicator::unchecked {width: 32px;height: 18px;image:url(&quot;:/new/prefix1/source/buttons/generalButton/radioNormal.png&quot;);position:relative;left: 0px;right: 5px;top: 2px;bottom:4px;}
QRadioButton::indicator::checked {width: 32px;height: 18px;image:url(&quot;:/new/prefix1/source/buttons/generalButton/radioSelected.png&quot;);position:relative;left: 0px;right: 5px;top: 2px;bottom:4px;}</string>
   </property>
   <property name="text">
    <string>小指</string>
   </property>
   <property name="checked">
    <bool>true</bool>
   </property>
   <property name="autoExclusive">
    <bool>false</bool>
   </property>
  </widget>
  <widget class="QPushButton" name="confirm_Btn">
   <property name="geometry">
    <rect>
     <x>1286</x>
     <y>700</y>
     <width>164</width>
     <height>70</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>黑体</family>
     <pointsize>25</pointsize>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">QPushButton{background: #0D9DDB;border-radius: 8px;color:white;}
QPushButton:hover{background: #1EAACF;border-radius: 8px;color:#333333;}</string>
   </property>
   <property name="text">
    <string>确定</string>
   </property>
  </widget>
  <widget class="QGroupBox" name="groupBox">
   <property name="geometry">
    <rect>
     <x>510</x>
     <y>530</y>
     <width>461</width>
     <height>201</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">#groupBox{border:none;}</string>
   </property>
   <property name="title">
    <string/>
   </property>
   <widget class="QPushButton" name="bucklingLevel3_Btn">
    <property name="geometry">
     <rect>
      <x>370</x>
      <y>20</y>
      <width>60</width>
      <height>60</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton{border-image: url(:/new/prefix1/source/buttons/level/level3NoSelect.png);}
QPushButton:hover{border-image: url(:/new/prefix1/source/buttons/level/level3Hover.png);}</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QPushButton" name="stretchLevel2_Btn">
    <property name="geometry">
     <rect>
      <x>280</x>
      <y>110</y>
      <width>60</width>
      <height>60</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton{border-image: url(:/new/prefix1/source/buttons/level/level2NoSelect.png);}
QPushButton:hover{border-image: url(:/new/prefix1/source/buttons/level/level2Hover.png);}</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QPushButton" name="stretchLevel3_Btn">
    <property name="geometry">
     <rect>
      <x>370</x>
      <y>110</y>
      <width>60</width>
      <height>60</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton{border-image: url(:/new/prefix1/source/buttons/level/level3NoSelect.png);}
QPushButton:hover{border-image: url(:/new/prefix1/source/buttons/level/level3Hover.png);}</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QPushButton" name="stretchLevel1_Btn">
    <property name="geometry">
     <rect>
      <x>190</x>
      <y>110</y>
      <width>60</width>
      <height>60</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton{border-image: url(:/new/prefix1/source/buttons/level/level1NoSelect.png);}
QPushButton:hover{border-image: url(:/new/prefix1/source/buttons/level/level1Hover.png);}</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QPushButton" name="bucklingLevel1_Btn">
    <property name="geometry">
     <rect>
      <x>190</x>
      <y>20</y>
      <width>60</width>
      <height>60</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton{border-image: url(:/new/prefix1/source/buttons/level/level1NoSelect.png);}
QPushButton:hover{border-image: url(:/new/prefix1/source/buttons/level/level1Hover.png);}</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="label_2">
    <property name="geometry">
     <rect>
      <x>-10</x>
      <y>35</y>
      <width>181</width>
      <height>30</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>黑体</family>
      <pointsize>20</pointsize>
     </font>
    </property>
    <property name="text">
     <string>屈曲阻力</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
   <widget class="QPushButton" name="bucklingLevel2_Btn">
    <property name="geometry">
     <rect>
      <x>280</x>
      <y>20</y>
      <width>60</width>
      <height>60</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton{border-image: url(:/new/prefix1/source/buttons/level/level2NoSelect.png);}
QPushButton:hover{border-image: url(:/new/prefix1/source/buttons/level/level2Hover.png);}</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="label_3">
    <property name="geometry">
     <rect>
      <x>-1</x>
      <y>125</y>
      <width>171</width>
      <height>30</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>黑体</family>
      <pointsize>20</pointsize>
     </font>
    </property>
    <property name="text">
     <string>伸展阻力</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
  </widget>
  <widget class="QLabel" name="label_25">
   <property name="geometry">
    <rect>
     <x>810</x>
     <y>190</y>
     <width>61</width>
     <height>16</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>黑体</family>
     <pointsize>10</pointsize>
    </font>
   </property>
   <property name="text">
    <string>位置</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QLabel" name="label_24">
   <property name="geometry">
    <rect>
     <x>700</x>
     <y>190</y>
     <width>61</width>
     <height>16</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>黑体</family>
     <pointsize>10</pointsize>
    </font>
   </property>
   <property name="text">
    <string>力量</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
